composer.json

```diff
@@ -1,20 +1,29 @@
 {
-    "name" : "bernard/normalt",
-    "license" : "MIT",
-
-    "autoload" : {
-        "psr-4" : { "Normalt\\" : "src" }
+    "name": "bernard/normalt",
+    "description": "Normalt is a extension to Symfony Serializer that only implements the Normalization part",
+    "keywords": ["normalization", "denormalization"],
+    "license": "MIT",
+    "require": {
+        "symfony/serializer": "^2.3 || ^3.0"
     },
-
-    "autoload-dev" : {
-        "psr-4" : { "Fixtures\\" : "spec/Fixtures" }
+    "require-dev": {
+        "doctrine/common": "^2.1",
+        "phpspec/phpspec": "^2.5",
+        "henrikbjorn/phpspec-code-coverage": "^2.0"
     },
-
-    "require" : {
-        "symfony/serializer" : "~2.3 || ~3.0"
+    "autoload": {
+        "psr-4": { "Normalt\\": "src/" }
     },
-
-    "require-dev" : {
-        "doctrine/common" : "~2.1"
+    "autoload-dev": {
+        "psr-4": { "Fixtures\\": "spec/Fixtures/" }
+    },
+    "scripts": {
+        "test": "vendor/bin/phpspec",
+        "test-ci": "vendor/bin/phpspec -c phpspec.yml.ci"
```

Comment:
yes, .dist indicates it is not a final configuration file, however the `ci` is a complete configuration and because of that i think it should keep it's `.yml` extension.